draft-terbu-sd-jwt-vc.md

```diff
@@ -175,72 +198,71 @@ The following is a non-normative example of a decoded SD-JWT header:
 }
 ```
 
-### Payload
-
-VC-SD-JWTs and VP-SD-JWTs can use any
-claim registered in the "JSON Web Token Claims" registry as defined in
-[@!RFC7519].
-
-Some of the claims in a VC MUST NOT be selectively disclosed as they are
-always required for processing on the verifier side. All other claims can be
-made selectively disclosable by the issuer when issuing the respective
-VC-SD-JWT.
-
-VC-SD-JWTs and VP-SD-JWTs MAY contain additional
-registered or public claims depending on the application.
-
-#### `type` claim {#type-claim}
-
-This specification defines the JWT claim `type`. The `type` claim is used
-to express the type of the JSON object that is secured by the
-JWT. The `type` value MUST be a case-sensitive `StringOrURI` value.
+### Claims
 
-The following is a non-normative example of how `type` is used to express
-a type:
+This section defines the claims that can be included in the payload of SD-JWTs
```

Comment:
Not sure I understand the rationale of this proposed change. It seems to mix the claims that "can" be used and the claims, where this draft defines requirements (a lot of then "MUST").  For the time being, I like the previous approach better.

Reply:
It was not clear in the previous version where to find those claims. Technically speaking, they are not always in the SD-JWT but sometimes in the Disclosure object. That is why this PR tried to adjust that language.

Reply:
we have a term for whole thing already -> VC-SD-JWT -> pls see terminology section. Same as VP-SD-JWT. Does this solve your issue?